lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -319,7 +320,7 @@ export class ResponseHandler {
             expiresOn = new Date(Number(cacheRecord.accessToken.expiresOn) * 1000);
             extExpiresOn = new Date(Number(cacheRecord.accessToken.extendedExpiresOn) * 1000);
         } else {
-            throw ClientAuthError.createAccessTokenEntityNullError();
+            expiresOn = new Date(0);
```

Comment:
Please add a logger warning message here.

Reply:
@jasonnutter this is a static method in ResponseHandler. The logger isn't directly accessible. I'm concerned the code change necessary will turn this into a bigger PR.  Although I don't mind doing it at all, do we think the overhead is worth it?

Reply:
That's fine, can be addressed later.